### ISSUE TITLE
Move 'fbjs' dependency to devDependencies

### ISSUE THREAD

**Issue by EECOLOR** [facebook/draft-js#2003]
The 'fbjs' dependency seems unused in the draft code itself. I think it would be nice to remove it as it contains a few dependencies. Having a leaner npm package seems to me like a win.

On top of that, it includes a polyfill for `fetch` which we include using a different style; only for browsers that need it (using a service).

**Comment by claudiopro** [comment]
> The 'fbjs' dependency seems unused in the draft code itself.

That is incorrect. Draft.js uses a number of utilities under the `/node_modules/fbjs/lib` directory, to name a few: `joinClasses`, `nullthrows`, `cx` etc.

**Comment by EECOLOR** [comment]
Ahh, I see. Is there an actual need for the `fetch` dependency? If not, is it at all possible to remove those (and other polyfill) dependencies?

I am not sure what the policy is for facebook and `fbjs`.
